src/apps/service/apps.clj

```diff
@@ -1,79 +1,21 @@
 (ns apps.service.apps
   (:use [apps.constants :only [de-system-id]]
-        [apps.service.oauth :only [authorization-uri has-access-token]]
+        [apps.service.apps-client :only [get-apps-client get-apps-client-for-username]]
```

Comment:
The code to create an apps client for the current user has been moved to a new namespace in order to make it easy to restart an asynchronous task if we ever decide we want to do that.